5.3/5.3.5/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.3.5
+Создать объект со следующей логикой:
+Запрашивает у пользователя любой текст: "Введите любой текст".
+А потом с помощью алерта выводит информацию об этом тексте (учитывай перенос строк в алерте), например:
+Введенный текст: "Абра ка табра";
+Всего символов: 13;
+Всего слов: 3.
+<script>
+	const infoObj = {
```

Comment:
This object has only 1 method. It would be more correct to divide into several methods (albeit the simplest ones) that will perform separate tasks:
1) a method that will ask the user for text
2) a method that will return the number of characters
3) a method that will return the number of words
4) a method that will show a message with all the information

We can discuss together